docs/generators/dart-dio.md

```diff
@@ -11,7 +11,7 @@ title: Documentation for the dart-dio Generator
 | generator type | CLIENT | |
 | generator language | Dart | |
 | generator default templating engine | mustache | |
-| helpTxt | Generates a Dart Dio client library with null-safety. | |
+| helpTxt | Generates a Dart Dio(X) client library with null-safety. | |
```

Comment:
remove this change since it's now only one dio

Reply:
Fixed, also removed the null-safety part, this seems obvious now with Dart 3 coming.